.builder/actions/build_samples.py

```diff
@@ -7,9 +7,7 @@
 class BuildSamples(Builder.Action):
     def run(self, env):
         # parse extra cmake configs
-        parser = argparse.ArgumentParser()
-        parser.add_argument('--cmake-extra', action='append', default=[])
-        cmd_args = parser.parse_known_args(env.args.args)[0]
+        cmd_args = env.args
```

Comment:
In PR https://github.com/awslabs/aws-crt-builder/pull/212, the aws-crt-builder no longer retrieve the arguments from the command-line, instead the builder would retrieve arguments from `env` variable. Fix the script to support latest builder.